project.clj

```diff
@@ -6,17 +6,31 @@
   :profiles {:dev {:dependencies [[org.clojure/clojure "1.10.0"]
                                   [criterium "0.4.4"]
                                   [cloverage "1.0.13"]
-                                  [fudje "0.9.7"]]
+                                  [fudje "0.9.7"]
+                                  [pjstadig/humane-test-output "0.9.0"]
+                                  ;; test-time dependencies
+                                  [org.clojure/core.async "0.4.490"]
```

Comment:
If I understand this correctly, these namespaces are needed only because the completion is tested against them. This is confusing. I would rewrite the tests so that they are tested against Clojure namespaces.

Reply:
Some things were tested against core.async historically. Removing will take time so I will do at the end